setup.py fails with py2exe

I've got this error from setup.py:

```
Traceback (most recent call last):
  File "setup.py", line 526, in <module>
    standardsetup("translate-toolkit", translateversion)
  File "setup.py", line 486, in standardsetup
    translatebashscripts)
  File "setup.py", line 490, in dosetup
    from setuptools import setup
  File "C:\Python27\lib\site-packages\setuptools\__init__.py", line 20, in <module>
    from setuptools.dist import Distribution, Feature
  File "C:\Python27\lib\site-packages\setuptools\dist.py", line 336, in <module>
    _Distribution = get_unpatched(distutils.core.Distribution)
  File "C:\Python27\lib\site-packages\setuptools\monkey.py", line 44, in get_unpatched
    return lookup(item)
  File "C:\Python27\lib\site-packages\setuptools\monkey.py", line 61, in get_unpatched_class
    raise AssertionError(msg)
AssertionError: distutils has already been patched by <class py2exe.Distribution at 0x047BBF48>
```

--- a/setup.py
+++ b/setup.py
@@ -22,6 +22,7 @@ import re
 import sys
 from distutils.sysconfig import get_python_lib
 from os.path import dirname, isfile, join
+from setuptools import setup
 
 try:
     from sphinx.setup_command import BuildDoc
@@ -487,7 +488,6 @@ def standardsetup(name, version, custompackages=[], customdatafiles=[]):
 
 
 def dosetup(name, version, packages, datafiles, scripts, ext_modules=[]):
-    from setuptools import setup
     description, long_description = __doc__.split("\n", 1)
     kwargs = {}
     if py2exe:

